api/types/crds/skupper_link_access_crd.yaml

```diff
@@ -28,8 +28,14 @@ spec:
                   type: string
                 tlsCredentials:
                   type: string
-                accessType:
-                  type: string
+                bindHosts:
+                  type: array
```

Comment:
Should this be an array or just a single value? Where we want to bind on multiple interfaces, would that not be via separate LinkAccess instances? One part of the role of this resource is to control the listeners in router config. I believe those can only specify one address to bind on? If so, then I think it is nicer to have a separate LinkACcess CR for each distinct address to bind to. (That also allows greater clarity on what names resolve to that interface).

Reply:
I was actually in doubt about it as well. Thinking of eventual usages on kube sites, I believe it makes more
sense to have a single host. I will update it.